tests/read_test.h

```diff
@@ -40,8 +38,9 @@ bool check_fixed_width_value(FixedWidthT actual, FixedWidthT expected, const std
 
 void processEvent(const podio::Frame& event, int eventNum, podio::version::Version fileVersion) {
   const float evtWeight = event.getParameter<float>("UserEventWeight").value();
-  if (evtWeight != (float)100. * eventNum) {
-    std::cout << " read UserEventWeight: " << evtWeight << " - expected : " << (float)100. * eventNum << std::endl;
+  if (evtWeight != static_cast<float>(100. * eventNum)) {
+    std::cout << " read UserEventWeight: " << evtWeight << " - expected : " << static_cast<float>(100. * eventNum)
```

Comment:
```suggestion
  if (evtWeight != 100.f * eventNum) {
    std::cout << " read UserEventWeight: " << evtWeight << " - expected : " << 100.f * eventNum)
```
A bit less noisy with the same effect